R/dbBind_AdbiResult.R

```diff
@@ -3,7 +3,7 @@
 #' @usage NULL
 dbBind_AdbiResult <- function(res, params, ...) {
 
-  if (!isFALSE(meta(res, "immediate"))) {
+  if (isTRUE(meta(res, "immediate"))) {
```

Comment:
`isTRUE(x) != !isFALSE(x)` for example when `x <- NULL`. Why the proposed change? `isFALSE()` was added in 3.5.0 I believe. The `Depends` setting of `R (>= 3.0.0)` should be adjusted to account for that. Alternatively we can always do `identical(x, FALSE)`, if backwards compatibility is of concern. But honestly, I'd rather bump the min R dependency a bit, as we currently only test from 3.6.3 onwards. What do you think?